config/locales/fr.yml

```diff
@@ -1,90 +1,106 @@
-      title: "Ensure patient has installed the COVID Shield app"
-      subheading: "Instruct the patient to:"
-      step1_html: "Visit <strong>http://gc.ca/covid-shield</strong>"
-      step2: "Click on either the Apple AppStore or GooglePlay button"
-      step3: "Install the app"
+      criteria_intro: "Les codes COVID Shield devraient seulement être donnés aux patients avec:"
+      criteria1: "un résultat confirmé de test positif à la COVID-19"
+      criteria2: "{autres critères additionnels optionels, tel que d'avoir un diagnostic confirmé}"
+      hide: "Voir moins"
+      show: "Voir plus"
     card2:
-      title: "Provide patient with COVID Shield code"
-      subheading: "Instruct the patient to:"
-      step1: "Open the COVID Shield app"
-      step2: "Tap on the white section at the bottom of the screen"
-      step2_modal_title: "Step 2"
-      step2_modal_content: "Tap on the white section at the bottom of the screen."
-      step2_modal_alt: "Screenshot of COVID Shield mobile app with bottom sheet highlighted"
-      step3_html: 'Tap the <code class="inline-code">Submit code</code> button'
-      step3_modal_title: "Step 3"
-      step3_modal_content_html: 'Tap the <span class="inline-code">Submit code</span> button.'
-      step3_modal_alt: "Screenshot of COVID Shield mobile app with submit code button highlighted"
-      step4: "In the code field, enter:"
-      step4_code_expired: "This code has expired."
-      step4_generate_code: "Generate code"
-      step4_generated_code: "Generated code"
-      step4_generate_new_code: "Generate new code"
-      step5_html: 'Tap the <code class="inline-code">I agree</code> button'
-      step5_modal_title: "Step 5"
-      step5_modal_content_html: 'Tap the <span class="inline-code">I agree</span> button.'
-      step5_modal_alt: "Screenshot of COVID Shield mobile app with I agree button highlighted"
+      title: "Assurez-vous que le patient a l'application COVID Shield d'installée sur son téléphone"
+      subheading: "Dites au patient de:"
+      step1_html: "Visiter <strong>https://www.covidshield.app/download</strong>."
+      step2: "Appuyer sur l'icône de l'App store d'Apple (iPhones) ou de Google Play (Android)."
+      step3: "Installer l'application."
+    card3:
+      title: "Fournir au patient un code COVID Shield"
+      subheading: "Dites au patient de:"
+      step1: "Appuyer sur l'icône pour ouvrir l'application COVID Shield."
+      step2: "Appuyer sur la section blanche au bas de l'écran."
+      step2_modal_title: "Étape 2"
+      step2_modal_content: "Appuyer sur la section blanche au bas de l'écran."
+      step2_modal_alt: "Capture d'écran de l'application mobile COVID Shield avec la section blanche au bas de l'écran encadrée"
+      step3_html: 'Appuyer sur le bouton <code class="inline-code">Entrez le code</code>.'
+      step3_modal_title: "Étape 3"
+      step3_modal_content_html: 'Appuyer sur le bouton <span class="inline-code">Entrez le code</span> .'
```

Comment:
space after </span> and before period

Reply:
There's only a space before the period. I removed it!